macro/src/lib.rs

```diff
@@ -1117,14 +1087,13 @@ mod tests {
                 }
             })
             .to_token_stream()
-            .to_string()
         ));
     }
 
     #[test]
     fn both_kinds_of_option() {
-        insta::assert_snapshot!(rustfmt_code(
-            &gen_bridge(parse_quote! {
+        insta::assert_snapshot!(pretty_print_code(
```

Comment:
thought (followup): we should take the whole `assert(pretty(gen(parse(...))).to_token_stream())` thing and turn it into an internal util macro